pkg/awsapi/eks.go

```diff
@@ -44,7 +44,7 @@ type EKS interface {
 	// that is created for your cluster. In most cases, it takes several minutes to
 	// create a cluster. After you create an Amazon EKS cluster, you must configure
 	// your Kubernetes tooling to communicate with the API server and launch nodes into
-	// your cluster. For more information, see Managing Cluster Authentication (https://docs.aws.amazon.com/eks/latest/userguide/managing-auth.html)
+	// your cluster. For more information, see Managing Cluster Authentication (https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html)
```

Comment:
please remove the changes here, its an autogenerated file.

Reply:
Noticed it was fixed by https://github.com/eksctl-io/eksctl/pull/7312/commits/9c9f66fab6995677eae17f9b9b5e4a2eeddfd52b

PR updated.